_posts/2022-06-13-mfa-phase-1-rollout.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Making popular Ruby packages more secure
+layout: post
+author: Betty Li
+author_email: betty.li@shopify.com
+---
+
+Attacks on the software supply chain are increasing. Our community has not gone unscathed. RubyGems has been affected by supply chain attacks in the past. It’s important for us to mitigate these risks as much as possible. Recommending stronger security practices like enabling multi-factor authentication (MFA) on popular packages is a first step towards improving security of the RubyGems ecosystem.
+
+Account takeovers are the [second most common](https://arxiv.org/abs/2002.01139) attack on software supply chains. The countermeasure against this type of attack is simple: enabling MFA. Doing so will help [reduce the chances](https://www.microsoft.com/security/blog/2019/08/20/one-simple-action-you-can-take-to-prevent-99-9-percent-of-account-attacks/) of a successful attack by 99.9 percent.
```

Comment:
nit: is reducing the chance of an attack by 99% the same as preventing 99% of attacks?

If so, nvm!